tools/oozie/oozie_simple_test/setup.sh

```diff
@@ -0,0 +1,3 @@
+sudo hdfs dfs -mkdir -p /tmp/oozie/shell-hadoopsh/
+sudo hdfs dfs -copyFromLocal job.properties /tmp/oozie/shell-hadoopsh/
+sudo hdfs dfs -copyFromLocal workflow.xml /tmp/oozie/shell-hadoopsh/
```

Comment:
This `sudo` is incorrect; I originally read this as `sudo -u hdfs` which would work to upload data as the HDFS user on machines which are running HDFS daemons such as namenodes.